content/scalers/apache-kafka-topic.md

```diff
@@ -132,4 +134,59 @@ spec:
       lagThreshold: "50"
     authenticationRef:
       name: keda-trigger-auth-kafka-credential
+```
+
+Your kafka head on Azure Event Hubs - turn on sasl ssl plain auth
```

Comment:
It should be a more generic example about SASL_SSL_PLAIN not specific to AEH.
This new added mechanism works with any Kafka "installation" not just AEH. From this doc, it could be misled imho.

Reply:
Agreed, I'd delete this line here or new section as mentioned below.

Reply:
point taken! Thanks for the feedback :)